README.md

```diff
@@ -1061,6 +1061,36 @@ resource "aws_instance" "main" {
 }
 ```
 
+###### Rename class.
+
+Try to rename a class:
+
+```cpp file=example.cpp
```

Comment:
Perhaps you can find a more creative example (and file name) 😄 An operation like "rename class" can already be performed very well using IDEs. `srgn` is more meant for things IDEs, `sed` etc. cannot already easily perform.

Reply:
Tried to make a more useful example. I did the best I could :laughing: